c7n/tags.py

```diff
@@ -1,11 +1,11 @@
 # Copyright The Cloud Custodian Authors.
 # SPDX-License-Identifier: Apache-2.0
 """
-Generic EC2 Resource Tag / Filters and actions
+Generic Resource Tag / Filters and Actions wrapper
 
 These work for the whole family of resources associated
 to ec2 (subnets, vpc, security-groups, volumes, instances,
-snapshots).
+snapshots) and resources that suport Amazon's Resource Groups Tagging API
```

Comment:
small typo here s/suport/support

Reply:
thanks for catching this, new commit below adds a "p"